packages/manager-components/src/components/templates/update-name-modal/translations/translations.ts

```diff
@@ -0,0 +1,27 @@
+import i18next from 'i18next';
+
+// import de_DE from './Messages_de_DE.json';
+// import en_GB from './Messages_en_GB.json';
+// import es_ES from './Messages_es_ES.json';
+// import fr_CA from './Messages_fr_CA.json';
```

Comment:
Is it normal those lines are commented ?

Reply:
I have to ask for translations, the files don't exist yet